include/crocoddyl/multibody/contacts/contact-1d.hpp

```diff
@@ -40,31 +40,52 @@ class ContactModel1DTpl : public ContactModelAbstractTpl<_Scalar> {
   /**
    * @brief Initialize the 1d contact model
    *
+   * To learn more about the computation of the contact derivatives in different
+   * frames see https://hal.science/hal-03758989/document.
```

Comment:
Instead of writing the HAL link, could you write the title of the paper?
Moreover, could you do this for the contact 3d and 6d classes?